### DISCUSSION TITLE
Can't run Ubuntu 18.04 in Apple Silicon.

### DISCUSSION THREAD

**Discussion by Junbal** [utmapp/UTM#3745] [General]
Hi- I'm trying to run Ubuntu 18.04 in M1. I did it with x86 but it's too slow. 
So, I try to do it with ARM.  I download it but if i select "Install Ubuntu server",
The display looks like this.

<img width="800" alt="스크린샷 2022-03-11 오후 12 34 28" src="https://user-images.githubusercontent.com/96059603/157797101-fa7e210d-5304-402f-9856-54ff884f41fc.png">

**Comment by titaniumtails** [comment]
What happens when you press [Enter] after this?

**Comment by clayton256** [comment]
I was able to install ARM Ubuntu 18.04 by selecting 'Console Only' under 'Display'.

**Comment by titaniumtails** [comment]
You might be able to switch it back to graphics, now you have done the install.
